packages/atomic/cypress/integration/facets-v1/facet-common-assertions.ts

```diff
@@ -0,0 +1,134 @@
+import {TestFixture} from '../../fixtures/test-fixture';
+import {ComponentSelector, should} from '../common-assertions';
+
+export interface FacetsSelector extends ComponentSelector {
```

Comment:
```suggestion
export interface BaseFacetSelector extends ComponentSelector {
```
I don't think the plural form is right here